app/helpers/zones_helper.rb

```diff
@@ -0,0 +1,30 @@
+module ZonesHelper
+  private
+
+  def settings_zone_list(zones, my_zone_name)
+    {
+      :headers  => [{:header => _("Zones"), :key => 'zone_type'}],
+      :rows     => settings_zone_rows(zones),
+      :selected => selected_zone(zones, my_zone_name)
+    }
+  end
+
+  def selected_zone(zones, my_zone_name)
+    !zones.empty? ? zones.where('name = ?', my_zone_name)&.first&.id&.to_s : []
+  end
+
+  def zone_name(my_zone_name, zone)
+    model = ui_lookup(:model => zone.class.to_s)
+    my_zone_name == zone.name ? "#{model} : #{zone.description}" + _(" (current)") : "#{model} \: #{zone.description}"
```

Comment:
Minor, but the parenthesis probably shouldn't be part of the translated part.  Also, we should use interpolation over `+`. Additionally, there's a bit of redundancy, so I think the following might be cleaner:

```suggestion
    name = "#{model} : #{zone.description}"
    name << " (#{_("current")})" if my_zone_name == zone.name
    name
```